src/Config.js

```diff
@@ -100,6 +106,17 @@ module.exports = class extends ChainedMap {
               .forEach(name => this.plugin(name).merge(value[name]));
           }
 
+          case 'plugins': {
+            if (Array.isArray(value)) {
```

Comment:
This seems to only handle the case when plugins is an array. Is that intended? Is this supposed to fall through to the next case if it's not an array?